components/ws-daemon/pkg/iws/iws.go

```diff
@@ -254,7 +254,8 @@ func (wbs *InWorkspaceServiceServer) PrepareForUserNS(ctx context.Context, req *
 		}
 
 		return &api.PrepareForUserNSResponse{
```

Comment:
We should force this fsshift type when the workspace is FWB. This way we need to only run fusefs for non-fwb workspaces.